shell/common/pipeline.h

```diff
@@ -116,8 +116,10 @@ class Pipeline {
 
     ProducerContinuation(const Continuation& continuation, size_t trace_id)
         : continuation_(continuation), trace_id_(trace_id) {
+      TRACE_EVENT_ASYNC_BEGIN0_WITH_FLOW_IDS("flutter", "PipelineItem",
+                                             trace_id_, /*flow_id_count=*/1,
+                                             /*flow_ids=*/&trace_id_);
       TRACE_FLOW_BEGIN("flutter", "PipelineItem", trace_id_);
```

Comment:
This doesn't do anything because flow events don't bind to async events in Chrome's trace format.